### PR TITLE
Get capacity provider from cluster. Don't hard-code anymore

### PR DESCRIPTION
**Pull request by yakloinsteak**

### CODE REVIEW

**Comment by yakloinsteak** [comment]
The second commit I think will help with the problems we have when we deploy all of production at once

File: config/deploy/docker/deploy [hunk 1]
```diff
@@ -8,9 +8,12 @@ args = CommandArgs.new
 args.deployments.each do |deployment|
   puts "Deploying #{deployment[:target_group_name]}..."
   Deployer.new(**deployment).run!
-end
 
-EcsTools.new.poll_state_until_stable!(CommandArgs.cluster)
+  # poll between deployments to prevent scaling out too much while we have
+  # double the web containers.
+  # i.e. don't proceed until it has stabilized
+  EcsTools.new.poll_state_until_stable!(CommandArgs.cluster)
```
Comment (eanders) [inline config/deploy/docker/deploy:15]: Does this mean it will deploy one app, then wait for it to stabilize, then deploy the next, etc.?

Comment (yakloinsteak) [reply config/deploy/docker/deploy:15]: Yes. I have a feeling not waiting is related to those times deploys wouldn't go through. We can pull this out if you want.

Comment (eanders) [reply config/deploy/docker/deploy:15]: Let's give it a try.  I'm all for waiting a bit if it means not having to spin up a ton of EC2 instances.

**Comment by yakloinsteak** [comment]
This is deployed to staging warehouse

**Pull request merged**